README.md

```diff
@@ -152,57 +55,143 @@ own [Contributing Guidelines][].
 [Meeting notes][] from past meetings are maintained for later reference.
 [Query issues for "Meeting"][] to find future meetings.
 
-### Summary / Current Status
-
-Our current focus is on site development. Development is happening in the [nodejs.dev repo](https://github.com/nodejs/nodejs.dev). This repo continues to be the hub for the redesign initiative.
-
 ### Team
 
-Any person who wants to contribute to the initiative is welcome! Please read
-[Contributing Guidelines][] and join the effort 🙌.
-
 This repo is managed by the nodejs.dev. Members of the nodejs website team
 also have commiter rights on the repo.
 
 #### The nodejs.dev team
 
```

Comment:
Apparantly github wont render svgs in addition to not letting you customize the css. The idea was to have all our avatar's instead of a text list.